clang/test/CodeGenSYCL/check-direct-attribute-propagation.cpp

```diff
@@ -146,15 +146,15 @@ class Functor10 {
 
 int main() {
   q.submit([&](handler &h) {
-    // CHECK: define {{.*}}spir_kernel void @{{.*}}kernel_name1() #0 !kernel_arg_buffer_location ![[NUM:[0-9]+]] !scheduler_target_fmax_mhz ![[NUM1:[0-9]+]]
+    // CHECK: define {{.*}}spir_kernel void @{{.*}}kernel_name1() #0 !kernel_arg_buffer_location ![[NUM:[0-9]+]] !kernel_arg_runtime_aligned ![[NUM:[0-9]+]] !scheduler_target_fmax_mhz ![[NUM1:[0-9]+]]
```

Comment:
Please add a comment what test is doing here? Thanks.

Reply:
I am not the original author of this test, but I can take a look  at it to see what it does and add a comment.

Reply:
I meant to add a comment for new metadata your Patch is creating

Reply:
Sorry I don't entirely follow your suggestion. 
Add a comment describing the metadata on every line I have modified in this test file? Or just this line?

Reply:
This test seems unchanged with your patch. You can ignore my comments here.